core/src/com/unciv/ui/worldscreen/WorldMapHolder.kt

```diff
@@ -87,8 +87,7 @@ class WorldMapHolder(internal val worldScreen: WorldScreen, internal val tileMap
                 }
 
                 override fun clicked(event: InputEvent?, x: Float, y: Float) {
-                    val unit = worldScreen.bottomUnitTable.selectedUnit
-                    if (unit == null) return
+                    val unit = worldScreen.bottomUnitTable.selectedUnit ?: return
```

Comment:
Also, please stop converting if checks to elvis operators. They're less immediately readable.

Reply:
But I find some files have used elvis operators.

Reply:
Chances are they were also put there by you... 🙃 
Elvis operators do have a limited use when they come as part of a longer chain that would otherwise require fiddly handling 
But for single 'if null return' checks, then 'if null return' is much more readable to the average coder

Reply:
I reedit it to original branch. In fact, when i find you merge someone's commit i think you change you mind, sorry😢